layouts/shortcodes/sitemap.html

```diff
@@ -0,0 +1,25 @@
+{{ $envURL := ( index .Site.Params.envURL (os.Getenv "BRANCH") ) }}
+{{ $envURL := cond (eq $envURL nil) .Site.Params.envURL.fed $envURL }}
+
+<div class="sitemap">
+  <h2>Main Navigation</h2>
+  <ul class="main-nav usa-list">
+      {{ $translation := ( index $.Site.Data.translations .Site.Language.Lang ) }}
+      <li><a hreflang="{{ $translation.Language.Lang }}" href="{{ .Site.Home.RelPermalink }}">Home</a></li>
```

Comment:
Replace "Home" with a variable and add it to the front matter in the md translations.